assets/app/view/game/company.rb

```diff
@@ -57,6 +57,8 @@ def render
             marginBottom: '0.5rem',
             fontSize: '90%',
           }
+          header_style[:background] = @company.header_color if @company.respond_to?(:header_color)
+          header_style[:color] = @company.header_text_color if @company.respond_to?(:header_text_color)
```

Comment:
Would it make sense to add header_color and header_text_color to company and default to the standard colors? That way we don't have to add any additional respond_to? nonsense, simplifying the code.

Reply:
Since im new here i dont know how much i can add to the "base" code. Its no problem to add it to the base company.

Reply:
Its definitely a judgement call and should be done carefully as it affects all games. My general rule of thumb is if the functionality would be useful to other games, I add it to the base class. If its only useful to my game, I add it to my specific game classes. In this case, I could see other games taking advantage of it. I'll probably use this for 1828 if it becomes available in the code base.

Reply:
remove this stuff

Reply:
Remove all the header_ code, so the minors and consession is the yellow color you mean @tobymao ?

Reply:
either make it all generic, or remove it, if you want to keep the PR simple, i'd remove all of this specialized stuff for now

Reply:
@tobymao, 1822 needs a way for the player to know who can use privates - majors only, players only, majors and minors. The print version uses the header color to distinguish. Seems like an unobtrusive way to distinguish here too. Just my two cents.

Reply:
Toby has beaten into my head the necessity to keep UI code out of the engine code. Putting a header_color method into company violates that. Better to have a method that indicates who may use a company and have the UI code look at that to decide on header colors.

Reply:
Im all for keeping ui code from the engine. But we have color on the major companies, just wanted it on the minors, concession to differ them during the bid. But i will remove it.

Reply:
if you need the color, then allow every game to have color and have it be a part of the config, don't have respond_to etc 

happy to chat more on slack if you want to discuss